### ISSUE TITLE
Display report expiration date changes on report page.

### ISSUE THREAD

**Issue by evilaliv3** [globaleaks/whistleblowing-software#4116]
### Proposal

This ticket is dedicated to the analysis and implementation for a functionality to show information about report expiration changes.

The idea is to extend the "Comment" section adding a comment for each report showing the following information
Author: The user that changed the expiration date
Date: The date of the operation
Text: "Expiration date: $ExpirationDateAfterChange

Example:

```
Author: Giorgio Fraschini
Date: June 12, 2024
Expiration date: June 12, 2025
```

### Motivation and context
Users should be able to see a track record of the changes to the expiration date of a report.

**Comment by evilaliv3** [comment]
To prepare for this feature implementation i'm proceeding addin an audit log entry  to keep track of the changes of the expiration date of a report.

**Comment by evilaliv3** [comment]
I think the we could keep this implementation uniform to the one implemented for #4115

**Comment by evilaliv3** [comment]
Closin the ticket since the feature is now implemented and released in version 5.
